src/main/java/net/minestom/server/utils/NBTUtils.java

```diff
@@ -174,11 +175,9 @@ public static void loadDataIntoItem(@NotNull ItemStack item, @NotNull NBTCompoun
                 if (attributeOperation == null) {
                     break;
                 }
+
                 final AttributeSlot attributeSlot = AttributeSlot.valueOf(slot.toUpperCase());
                 // Wrong attribute slot, stop here
-                if (attributeSlot == null) {
-                    break;
-                }
```

Comment:
what's the motivation of removing this case?

Reply:
Because atribute slot is never null, i changed that because the client by default dont send the attributeSlot

Reply:
It is here because you cannot trust the client